forest/src/daemon.rs

```diff
@@ -40,7 +42,21 @@ pub(super) async fn start(config: Config) {
         });
 
     // Initialize keystore
-    let mut ks = PersistentKeyStore::new(config.data_dir.to_string()).unwrap();
+    let passphrase = if let true = config.encrypt_keystore {
```

Comment:
You don't need the `if let`, almost reads like poetry though

Reply:
Oh, good point! I'm not sure I understand this usage of if let.

Reply:
Wait, is this syntactically valid??? 😮

Reply:
I wouldn't be surprised if it was, but I honestly couldn't say what the return value would be, or how the assignment works... It's kinda breaking my brain rn, ngl :joy_cat:

Reply:
Noted. Using a match instead.

Reply:
The if statement is fine, you just don't need to destructure the boolean since it has no ... structure :)

Reply:
> Wait, is this syntactically valid???

yes it is lol

Reply:
@olibearo ahhh, that's how to read it...